task9/index.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import {config} from "./config/config.js";
+import meteorRouter from './delivery/meteorController.js'
+
+const app = express();
+
+app.use(meteorRouter)
+
```

Comment:
add `errorHandler` please somewhere there

here you can find a little bit more information https://medium.com/@mohsinogen/simplified-guide-to-setting-up-a-global-error-handler-in-express-js-daf8dd640b69

Reply:
done